concordium-std/tests/derive-schema-type/success-transparent.rs

```diff
@@ -0,0 +1,26 @@
+/// Test ensuring derivation works for newtype wrappers with transparent
+use concordium_std::*;
+
+#[derive(SchemaType)]
+#[concordium(transparent)]
+struct MyStruct {
+    field: u32,
+}
```

Comment:
Please add a test with a container and `size_length`.

Reply:
Done